src/shared/schemas/evacuationAR.ts

```
@@ -0,0 +1,43 @@
+import { z } from 'zod';
```

ESLint

[error] Replace `'zod'` with `"zod"`

(prettier/prettier)